spec/requests/api/v1/bills/search_request_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe 'Bill Search Endpoints' do
   describe "bill search: bills found in database" do
-    it "returns data about specific bills based on search query params", :vcr do
+    xit "returns data about specific bills based on search query params", :vcr do
 
```

Comment:
same question about this "x"ed out test -- is it working? can we add it back in? If i recall, this was working previously, curious what happened?